src/Tizen.NUI/src/public/Animation/Animation.cs

```diff
@@ -679,6 +679,10 @@ internal static Animation DownCast(BaseHandle handle)
         /// </remark>
         /// <param name="action">The end action can be set.</param>
         /// <since_tizen> 3 </since_tizen>
+        /// <remarks>
```

Comment:
I don't think we need to repeat that information in remarks. BTW, the `<remark>` is above is not a valid tag.
Please fix lines 676 and 679: remark -> remarks.
If you want to keep that note: add it to existing remarks section, leaving empty comment line or inserting `<br/>` to separate paragraphs/break the line. And the information should be changed a little into something like "The method was rarely used and was deprecated since API11." or "The method is deprecated since API11 because it was not used."

Reply:
Hi. Thank you for your review point. Now I change the tag name as `remarks`, and also add more informations the another way to do the same behaviour.